Sources/Penny/EvolutionChecker.swift

```diff
@@ -57,6 +59,24 @@ actor EvolutionChecker {
             return
         }
 
+        let proposalIDsWithError = proposals.filter {
+            !($0.errors ?? []).isEmpty
+        }.map(\.id)
+        if !proposalIDsWithError.isEmpty {
+            let allAreAlreadyReported = proposalIDsWithError.allSatisfy {
+                self.reportedProposalIDsThatContainErrors.contains($0)
+            }
```

Comment:
```suggestion
            let allAreAlreadyReported = proposalIDsWithError.isSuperset(of: self.reportedProposalIDsThatContainErrors)
```

Reply:
```swift
self.reportedProposalIDsThatContainErrors
    .isSuperset(of: proposalIDsWithError)
```
is what we'd want here 🙂

Reply:
What fun is it if I do _all_ the work for you? It can't at all be that I made a mistake! Or that I meant `isSubsetOf`! Can't be! 🤣